spring-cloud-dataflow-core/pom.xml

```diff
@@ -28,10 +28,26 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-deployer-spi</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
We should try not to move things from registry to core package as we're been trying to keep deps tidy in core package.

Reply:
I didn't want to move that stuff around...but in order for the JSON serialization to work I didn't have a choice.  I'm open to other ideas.

Reply:
Can you explain why serialization wouldn't work?

Reply:
The ObjectMapper needs to have visibility to the classes that are being serialzied/deserialized.  In this case, embedded within the manifest are the `Resource` for the app.  That means we need access to those classes.

Reply:
Now that the `objectMapper` is customized in `spring-cloud-dataflow-server-core`, `ResourceDeserializer` could also be there instead in `spring-cloud-dataflow-core` where it sees everything. Thus you don't need to move this stuff around. Unless I missed something else wouldn't that work then.